.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2013-2020 the original author or authors from the JHipster project.
+#
+# This file is part of the JHipster project, see https://www.jhipster.tech/
+# for more information.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Application CI
+on: [push, pull_request]
+jobs:
+    pipeline:
+        name: jhipster-online pipeline
+        runs-on: ubuntu-latest
+        if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.pull_request.title, '[skip ci]') && !contains(github.pull_request.title, '[ci skip]')"
```

Comment:
When taking the pull request title; it should be; `github.event.pull_request.title`. We have changed this on the [other Github Actions](https://github.com/jhipster/generator-jhipster/blob/ec26d48bf7683ba768d4ff07dcfcf73e8f141fb1/.github/workflows/react.yml#L40) 😄